src/Tribe/Template/Month.php

```diff
@@ -1234,5 +1234,50 @@ public function has_events() {
 			return (bool) $this->events_in_month;
 		}
 
+		/**
+		 * Check if the month has events when all the filters have been applied.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool
+		 */
+		public function has_events_filtered() {
+
+			if ( 'month' === Tribe__Events__Main::instance()->displaying ) {
```

Comment:
Lets invert this logic, to `return false;` if `!==` than `month`, avoids indentation.